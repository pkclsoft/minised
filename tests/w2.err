print this to stderr, but don't include it in the output
