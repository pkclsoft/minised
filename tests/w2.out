don't print this to stderr
print this too, but *do* include it
print this too, but *do* include it
this is just filler
